nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -360,7 +361,7 @@ in {
 
       services.phpfpm = {
         phpOptions = phpOptionsExtensions;
```

Comment:
If you don't mind would you consider moving this line *into* the pool? See https://github.com/NixOS/nixpkgs/pull/55742 for details.

Reply:
@aanderse If I understand correctly both `phpOptions` and `phpPackage` can be moved to the pool right?

Reply:
Done, both moved to pool